Libraries/Image/RCTImageLoader.m

```diff
@@ -377,10 +371,24 @@ - (RCTImageLoaderCancellationBlock)loadImageOrDataWithURLRequest:(NSURLRequest *
                                                    userInfo:nil], nil);
           return;
         }
-      }
 
-      // Call handler
-      completionHandler(nil, data);
+        NSString *responseDate = ((NSHTTPURLResponse *)response).allHeaderFields[@"Date"];
```

Comment:
I think it is good point, because the Date header is not 'MUST' thing on [RFC2616](https://tools.ietf.org/html/rfc7231#section-7.1.1.2).